Automattic/wp-calypso
# Update Anti-spam card copy

### Changes proposed in this Pull Request

Update the copy of the Anti-spam card in the pricing page.

Fixes 1164141197617539-as-1199666226766814

### Implementation notes

New copy overwrites the copy of any previous version.

### Testing instructions

- Run the PR in Jetpack cloud or Calypso
- Visit the pricing/plans page
- Check the copy of the Anti-spam card against the copy linked in the ticket, for both monthly and yearly terms

### Screenshots

#### Before
<img width="359" alt="Screen Shot 2021-01-25 at 2 56 31 PM" src="https://user-images.githubusercontent.com/1620183/105766065-46231680-5f27-11eb-9f00-d0ab44d31912.png">

#### After
<img width="358" alt="Screen Shot 2021-01-25 at 3 04 58 PM" src="https://user-images.githubusercontent.com/1620183/105766076-48857080-5f27-11eb-8413-e7d9f0ed76ea.png">
